client/ctclient/ctclient.go

```diff
@@ -220,6 +220,13 @@ func getInclusionProof(ctx context.Context, logClient client.CheckLogClient) {
 			log.Fatalf("Failed to create hash of leaf: %v", err)
 		}
 		hash = leafHash[:]
+
+		// Print a warning if this timestamp is still within the MMD window
+		now := time.Now()
+		ts := time.Unix(0, *timestamp*1000000)
```

Comment:
For consistency with the rest of the file, could you use `when := ct.TimestampToTime(*timestamp)` ?

Reply:
done